specs-go/v2/artifacttype.go

```diff
@@ -0,0 +1,6 @@
+package v2
+
+const (
+	// ArtifactTypeNotaryV2 specifies the artifact type for a notary V2 object.
+	ArtifactTypeNotaryV2 = "application/vnd.cncf.notary.v2"

```

Comment:
Curious why we need a notary v2 const got here.
The oci.artifacts content shouldn't have anything about the various artifacts. It should only have the generics that support all types, and no specific types added.

Reply:
Cleaned up